node pool panic on missing arguments

**Bug description**
 node pools panic if theres no name set on the cluster,.... 
```
tanzu cluster node-pool set -f nodepool.yaml
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.runSetNodePool(0x57d5e60, 0xc00087ef20, 0x0, 0x2, 0x0, 0x0)
	/go/src/cayman-tanzu-framework/tanzu-framework/src/cmd/cli/plugin/cluster/set_node_pool.go:48 +0x136
github.com/spf13/cobra.(*Command).execute(0x57d5e60, 0xc00087ef00, 0x2, 0x2, 0x57d5e60, 0xc00087ef00)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc000628280, 0xc000db1f20, 0xb, 0xb)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/vmware-tanzu/tanzu-framework/pkg/v1/cli/command/plugin.(*Plugin).Execute(...)
	/go/src/cayman-tanzu-framework/tanzu-framework/src/pkg/v1/cli/command/plugin/plugin.go:54
main.main()
	/go/src/cayman-tanzu-framework/tanzu-framework/src/cmd/cli/plugin/cluster/main.go:56 +0x267
Error: exit status 2
``` 
**Expected behavior**

the cli should output "missing cluster name"

--- a/cmd/cli/plugin/cluster/set_node_pool.go
+++ b/cmd/cli/plugin/cluster/set_node_pool.go
@@ -27,6 +27,7 @@ var setNodePoolOptions clusterSetNodePoolCmdOptions
 var clusterSetNodePoolCmd = &cobra.Command{
 	Use:   "set CLUSTER_NAME",
 	Short: "Set node pool for cluster",
+	Args:  cobra.ExactArgs(1),
 	RunE:  runSetNodePool,
 }
 

